### PR TITLE
Re-implements old carpet sprite

### PR DESCRIPTION
**Pull request by pudl**
This readds an old carpet sprite from r4407 that could be used for more decoration on carpets. It's not THAT pretty, but it doesn't hurt to have it back again and someone might see a good use for it.
Also, I fixed some typos of the word "atheist".

### CODE REVIEW

**Comment by Jalleo** [comment]
@xxalpha this will need a new floor tile for mappers to use it properly

**Comment by xxalpha** [comment]
Yes, floor type please.

**Comment by pudl** [comment]
I dont need to give it a floor type, you just icon edit the carpet subtype, just like how you have to get the carpetsymbol. This is no different.

**Comment by xxalpha** [comment]
Then this PR is unofficially denied.

**Comment by RemieRichards** [comment]
officially denied too.
floor type or bust.

a refactor for this was not done just for funsies.

**Comment by pudl** [comment]
What the fuck, carpet symbols don't have a floor type, what makes this different. I'm just being consistent you shit.

**Comment by optimumtact** [comment]
Answer their question please @RemieRichards and @xxalpha

**Comment by optimumtact** [comment]
Answer their question or I really don't feel I can hold this pr up.

**Comment by RemieRichards** [comment]
It should be evident.

consistency is good, unless you're being consistent with something that's bad.
example: just because the other carpet symbols don't have types, doesn't mean this one can get away with it, they should all have types.

**Comment by xxalpha** [comment]
It's fine if you want to add an icon.

If you want to map it as a floor that's a different story.

**Comment by pudl** [comment]
Remove dat dnm

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 1]
```diff
@@ -80,6 +80,11 @@
 	burnt = 1
 	update_icon()
 
+/turf/open/floor/carpet/carpetsymbol
+	icon_state = "carpetsymbol"
+
+/turf/open/floor/carpet/carpetsymbol2
+	icon_state = "carpetsymbol2"
```
Comment (ChangelingRain) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: This won't work because they'll smooth and lose the icon immediately.

Comment (pudl) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: I originally added the SMOOTH_FALSE but I don't know anything about code so I was worried it wouldn't work. I'll add it now.

**Pull request merged**